### ISSUE TITLE
splitting title over line break in specific place

### ISSUE THREAD

**Issue by epageperron** [elotroalex/ed#37]
Hi! As a newbie with yaml and liquid (and jekyll..) 
Using a longer title than "Ed.", I couldn't figure out how to split my title where I wanted and not with the natural soft wrap. I found out the usage of "|-" in the yaml front matter, I also had to add the filter "newline_to_br" in the layout. I suggest adding the filter by default in the layout.

**Comment by elotroalex** [comment]
thanks for this! I added some css to break the long lines, but your solution is perfect. @epageperron , where did you add it exactly in the layout?

**Comment by epageperron** [comment]
It's line 22 in my _layouts/default.html 

`          <h3 class="masthead-title">`
`             <a href="{{ site.baseurl }}/" title="Home">{{ site.title | newline_to_br }}</a>`
`             <br><small>{{ site.tagline }}</small>`
 `         </h3>`

**Comment by elotroalex** [comment]
Perfect! I will add it and test it soon. What is your website that you're talking about here?

**Comment by elotroalex** [comment]
done!

**Comment by epageperron** [comment]
It was the website for a course I gave in the Fall 2016. I published one of the papers of my students on the site and they used hypothes.is to give  feedback to their colleagues and reply to the feedback. Unfortunately I changed my GitHub username since so the annotations are gone :(   https://epageperron.github.io/relane/  I know it wasn't the exact intended usage of your theme but it really served well!
